TS won't compile on meinberlin

```
src/current/current/build/js/Packages/Proposal/Proposal.ts(132,36):
error TS2339: Property 'address' does not exist on type '{ title:
string; detail: string; rateCount: number; creator: string;
creationDate: string; commen...'.
src/current/current/build/js/Packages/Proposal/Proposal.ts(132,61):
error TS2339: Property 'address' does not exist on type 'Sheet'.
```

--- a/src/meinberlin/meinberlin/static/js/Packages/Proposal/Proposal.ts
+++ b/src/meinberlin/meinberlin/static/js/Packages/Proposal/Proposal.ts
@@ -129,7 +129,6 @@ var bindPath = (
                     };
                     if (isKiezkasse) {
                         scope.data.budget = kiezkasseSheet.budget;
-                        scope.data.address = kiezkasseSheet.address;
                         scope.data.creatorParticipate = kiezkasseSheet.creator_participate;
                         scope.data.locationText = kiezkasseSheet.location_text;
                     } else if (isBuergerhaushalt) {

